src/state/reducers/statusReducer.ts

```diff
@@ -14,10 +14,10 @@ export default createReducer({}, (builder) => {
   builder
     .addMatcher(
       ({ type }) => type.endsWith(`/${actionStatus.REJECTED}`),
-      (state: any, { type, error }) => {
-        state[getActionKey(type)] = {
+      (state: any, action: any) => {
```

Comment:
Is there any chance of avoiding any here? Do you think you could provide a typing for this action?